src/nativestore/NodeManager.cpp

```diff
@@ -308,5 +414,5 @@ void NodeManager::setIndexKeySize(unsigned long newIndexKeySize) {
     this->INDEX_KEY_SIZE = newIndexKeySize;
 }
 unsigned int NodeManager::nextPropertyIndex = 0;
-std::string NodeManager::NODE_DB_PATH = "streamStore/g{}_p{}.db";
+std::string NodeManager::NODE_DB_PATH = "/home/ubuntu/software/jasminegraph/streamStore/g{}_p{}.db";
```

Comment:
Lets make this hard coded path  configuration parameter taken from config file

Reply:
Removing this line as it is not used. The actual nodesDBPath is constructed in lines 37 & 41.

Reply:
Ok